src/main/java/org/folio/service/orders/flows/update/unopen/UnOpenCompositeOrderManager.java

```diff
@@ -87,6 +87,10 @@ public Future<Void> process(CompositePurchaseOrder compPO, CompositePurchaseOrde
 
   }
 
+  public Future<Void> rollbackInventory(CompositePurchaseOrder compPO, RequestContext requestContext) {
+    return processInventory(compPO.getCompositePoLines(), true, requestContext);
```

Comment:
let's also format method processInventoryHoldingWithItems() in this class

Reply:
I fixed the indents.